docs/user-guides/federation.md

```diff
@@ -18,17 +18,20 @@ directory](federation/manifests)
 
 ### Create a Project to deploy MonitoringStack
 
-
 ```yaml
```

Comment:
maybe we can get rid of the snippets in here so we don't have to keep them synced with `docs/user-guides/federation/manifests`?

Reply:
ideally we could use something like mdox to inject the manifests into the doc but i didn't want to go too deep into the rabbit hole.